### PR TITLE
Update nf-libloaderapi-loadresource.md

### PR DESCRIPTION
**Pull request by asklar**
GlobalSize doesn't work for resource HGLOBALs

### CODE REVIEW

**Comment by PRMerger14** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/libloaderapi/nf-libloaderapi-loadresource.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ If the function fails, the return value is <b>NULL</b>. To get extended error in
 
 The return type of <b>LoadResource</b> is <b>HGLOBAL</b> for backward compatibility, not because the function returns a handle to a global memory block. Do not pass this handle to the <a href="/windows/desktop/api/winbase/nf-winbase-globallock">GlobalLock</a> or <a href="/windows/desktop/api/winbase/nf-winbase-globalfree">GlobalFree</a> function. To obtain a pointer to the first byte of the resource data, call the <a href="/windows/win32/api/libloaderapi/nf-libloaderapi-lockresource">LockResource</a> function; to obtain the size of the resource, call <a href="/windows/win32/api/libloaderapi/nf-libloaderapi-sizeofresource">SizeofResource</a>. 
 
+Note that [GlobalSize](/windows/desktop/api/winbase/nf-winbase-globalsize) will return 0 for a resource HGLOBAL. As a result, any APIs that depend on GlobalSize to determine the size of the HGLOBAL will not function correctly.
```
Comment (asklar) [inline sdk-api-src/content/libloaderapi/nf-libloaderapi-loadresource.md]:
```suggestion
Note that [GlobalSize](/windows/desktop/api/winbase/nf-winbase-globalsize) will return 0 for a resource HGLOBAL. As a result, any APIs that depend on GlobalSize to determine the size of the HGLOBAL will not function correctly.
For example, don't use [CreateStreamOnHGlobal](/windows/win32/api/combaseapi/nf-combaseapi-createstreamonhglobal), as it relies on it. Instead, use [SHCreateMemStream](/windows/win32/api/shlwapi/nf-shlwapi-shcreatememstream).
```

**Comment by PRMerger17** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Pull request merged**